stl/inc/generator

```diff
@@ -210,7 +210,7 @@ public:
     }
 
     _NODISCARD auto yield_value(const remove_reference_t<_Yielded>& _Val) noexcept(
-        is_nothrow_constructible_v<remove_cvref_t<_Yielded>, const remove_reference_t<_Yielded>&>)
+        is_nothrow_constructible_v<remove_cvref_t<_Yielded>, const remove_reference_t<_Yielded>&>) /* strengthened */

```

Comment:
Stylistic nitpick: We use `/* strengthened */` when it's embedded in a line (especially before an opening brace), but the following `requires` will always be clang-formatted on a new line, so this should ideally be a C++ comment `// strengthened`. Not worth resetting testing though.